src/modules/ROOT/pages/changelog/202303/07.json

```diff
@@ -0,0 +1,122 @@
+      ],
+      "why": [
+        "to make sorting/viewing data easier for the user."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1449/commits/e73119a888474ef397f837cea854d29b15cf2035"
+      ]
+    }
+  ],
+  
+  "changed": [
+    {
+      "what": [
+        "option to clone a page layout by adding two sub options - for cloning a page with page block reference and for cloning a page without page block reference.",
+        "In the first case, configuration changes that are made to the original page's blocks are also reflected to the copied page's page blocks.",
+        "As for the other option, the changes made to the original page's blocks are not reflected to the copied page's page blocks."
+      ],
+      "why": [
+        "to provide the user with the option to choose whether the copied page's block configuration should be shared among pages."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1444/commits/d64ed2b456cb0ee8cb064af817af48e06ad27c9b"
+      ]
+    },
+    {
+      "what": [
+        "the UI for uploading a namespace by showing a spinner while data is being processed and hiding the name and handle field of the namespace instead of showing the spinner and fields at the same time."
+      ],
+      "why": [
+        "to be consistent with the UI in Corteza."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/c997655b961965ee607907c54fd68ba10686fcbd"
+      ]
+    },
+    {
+      "what": [
+        "the handle field for a user to not be required when updating the resource in {APP_NAME_ADMIN}."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/ed377a7d2b792bd56497a18e7a5e0517e149afd7"
+      ]
+    },
+    {
+      "what": [
+        "general ui/ux improvements:",
+        "** Added additional information on how to write conditions for hiding or showing for a page layout in the page layout configurator.",
+        "** Updated the edit icon in page tree to match the same edit icon used everywhere in Corteza.",
+        "** Standardize permissions button in {APP_NAME_ADMIN} and {APP_NAME_COMPOSE}."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/d2cc2c8dc3ada68b19093ac99097734a9dd0ac85",
+        "https://github.com/cortezaproject/corteza/commit/d5f8faac5a19fa4a11a9509e7783e95773a5772b",
+        "https://github.com/cortezaproject/corteza/commit/86c12cc57cb83dc48a69ee36ea35c06a81e88f08"
+      ]
+    }
+  ],
+  "fixed": [
+    {
+      "what": [
+        "non-record page blocks not being refreshed in a modal after a record has been added or edited."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1444/commits/a544780753080173aad1bf03437eec836826c636"
+      ]
+    },
+    {
+      "what": [
+        "the `refresh all record values` prompt in workflows not updating record values when workflow is executed."
```

Comment:
Was there an error? If so, do point out the exact error

Reply:
There was no error. The records didn't update